extras/bindings/c/include/precice/SolverInterfaceC.h

```diff
@@ -356,19 +311,12 @@ void precicec_writeBlockVectorData(
  * @param[in] dataPosition Spatial position of the data to be written.
  * @param[in] dataValue Vectorial data value to be written.
  */
-void precicec_writeVectorData(
-    int           dataID,
-    int           valueIndex,
-    const double *dataValue);
+void precicec_writeVectorData(int dataID, int valueIndex, const double *dataValue);
 
 /**
  * @brief See precice::SolverInterface::writeBlockScalarData().
  */
-void precicec_writeBlockScalarData(
-    int           dataID,
-    int           size,
-    const int *   valueIndices,
-    const double *values);
+void precicec_writeBlockScalarData(int dataID, int size, const int *valueIndices, const double *values);
```

Comment:
With this changes, `clang-format` resolves the current formatting ambiguity of our code base (e.g. both code versions would have been accepted in our current formatting settings). However, having multiple function arguments on multiple lines might be desirable. Do we want to enforce something line breaks in function arguments or go with the current version?

Reply:
I prefer one item per line. I find that it makes the signature easier to digest.

Reply:
Yes I agree, these arguments
```
BinPackArguments: false
BinPackParameters: false
```
would set the arguments on separate lines if they don't fit into the 120 characters. Or do you want to enforce the separate line for all functions?